docs/defaulthooks.md

```diff
@@ -70,6 +69,30 @@ This works even for complex values, like tuples.
 'siamese'
 ```
 
+Enum structuring and unstructuring even works for complex values, like tuples, but if you have anything but simple literal types in those tuples (`str`, `bool`, `int`, `float`) you should consider defining the Enum value's type via the `_value_` attribute's type hint so that cattrs can properly structure it.
```

Comment:
Let's link https://typing.python.org/en/latest/spec/enums.html#member-values in here somewhere.